### DISCUSSION TITLE
第2节课笔记(22班)

### DISCUSSION THREAD

**Discussion by BarryYin** [InternLM/Tutorial#232] [笔记]
在这里提交链接

**Comment by cavedweller509** [comment]
https://github.com/BestAnHongjun/InternLM_Notes/tree/main

**Reply by cavedweller509** [reply]
更新：https://github.com/BestAnHongjun/InternLM_Notes/blob/main/notes/lecture_2.md

**Comment by Jasimyn** [comment]
https://broadleaf-gemini-274.notion.site/Lecture2-Demo-c69108bbac7e4223bd1f1a759f12495f

**Comment by jiayaozhang** [comment]
https://github.com/jiayaozhang/InternLM_Notes/blob/main/Lecture2.md

**Comment by liuwei16** [comment]
https://g5b4uupmby.feishu.cn/wiki/QFEKwUNrfi4bXukLAL9cDwEvnOc

**Comment by CDL0726** [comment]
https://github.com/CDL0726/InternLM_Notes/blob/main/README.md

**Comment by lilxmx** [comment]
https://blog.csdn.net/L1142456160/article/details/135491993

**Comment by MinghanJia** [comment]
https://github.com/MinghanJia/InternLM/blob/main/Lecture2.md

**Comment by aqizhoua** [comment]
https://aqizhoua.github.io/poster/89484759.html

**Comment by cyq2368** [comment]
https://mwsnr0a94gx.feishu.cn/docx/NUzAdHquIoApslxI1mOcLSiunQh?from=from_copylink

**Comment by lankuohsing** [comment]
https://github.com/lankuohsing/InternLM_notes/blob/main/lecture_2/note_lecture2.md

**Comment by Lynn401** [comment]
https://daisy-401.notion.site/Tutorial-2-Demo-70b1df5df1214462bd406836687510e4?pvs=4

**Comment by Juanting-Xu** [comment]
https://github.com/Juanting-Xu/InternLM_notes/blob/main/lecture2.md

**Comment by miku8miku** [comment]
https://note.youdao.com/s/RjmXjCOn

**Comment by hyanworkspace** [comment]
https://note.youdao.com/web/#/file/recent/note/WEB0b050e38d8c6c83bbee8ce15fe3e0112/

**Comment by linkch0** [comment]
https://linkch0.github.io/courses/InternLM/lecture/#l2-demo

**Comment by lyyilin** [comment]
https://blog.csdn.net/qq_73668945/article/details/135516255

**Comment by vjsmmk** [comment]
https://github.com/vjsmmk/puyu/blob/main/2/note

**Comment by miyc1996** [comment]
https://github.com/miyc1996/InternLM/blob/main/lesson2.ipynb

**Comment by funcfang** [comment]
https://www.yuque.com/funcfang/ke5qbq/vay3w1hx54sl1qk2?singleDoc#mVG4q

**Comment by siruzhong** [comment]
https://siru.notion.site/2-Demo-1b00299624aa4f1b8af5739912501d05?pvs=4

**Comment by leonfrank** [comment]
https://github.com/leonfrank/InternLM-HW/blob/main/lecture2/note2.pdf

**Comment by GradOpt** [comment]
https://github.com/OswaldHongyu/InternLM_notes/blob/main/lecture2.pdf

**Comment by TAICHIKF** [comment]
https://github.com/TAICHIKF/InternLM_notes/blob/main/2%20InternLM_Demo.pdf

**Comment by mmb135** [comment]
https://www.yuque.com/r/note/0d04f4e3-3fcc-4d93-a7e0-e7abafa3739b
